### ISSUE TITLE
The basic setup seems to work

### ISSUE THREAD

**Issue by Razpudding** [TuriGuilano/twitter-realtime#2]
I managed to get your app working as per instructions listed. I did notice the server crashing quite frequently. Something about

> Unexpected token E in JSON at position 0

Is that a known bug?

**Comment by TuriGuilano** [comment]
Nope.. I dont have this bug? Maybe you can show it to me lateron today!
